### ISSUE TITLE
Idea: show/hide entries in card based on rules/conditions

### ISSUE THREAD

**Issue by mbobrovs** [skogsaas/homey.dashboards#110]
This would enable e.g. to make a card that only shows error or bad conditions, as e.g.

- humidity in a room too high
- battery of a device almost empty
- door unlocked

or give an overview on what's currently going on in the house, as e.g.

- A/C running in a room
- PV power > 1000 W
- Heat Pump running

PS: adds perfectly to the idea to format/style entries based on rules/conditions

**Comment by skogsaas** [comment]
Agreed, this fits perfectly with existing ideas for formatting, I just hadn't thought about hiding widgets as a part of this 🤔

Though, hiding entire cards is unfortunate, as any errors in rules and conditions might try to display two cards at the same time at the same position. So this feature should be limited to widgets inside cards to begin with.

**Comment by mbobrovs** [comment]
by the way, I did not mean to hide entire cards, but e.g. to have a card called "Warnings", but to show/hide capabilities or insights based on rules.

Especially for capabilities: define multiple rules with that either can change formatting (e.g. color) or completely hide the entry - a little bit like conditional formatting in Excel, but also with option to hide entry completely
